types/redis/index.d.ts

```diff
@@ -43,7 +44,7 @@ export interface ClientOpts {
     password?: string;
     db?: string | number;
     family?: string;
-    rename_commands?: { [command: string]: string };
+    rename_commands?: { [command: string]: string } | null;
```

Comment:
`rename_commands` is already set optional. Why do we need this?

Reply:
my mistake, i was under the assumption we had `strictNullChecks` turned on